### ISSUE TITLE
RxJs 6 Support

### ISSUE THREAD

**Issue by iamwyza** [akserg/ng2-toasty#99]
* **I'm submitting a ...**
[x] bug report
[ ] feature request
[ ] question about the decisions made in the repository

* **Do you want to request a *feature* or report a *bug*?**

bug

* **What is the current behavior?**

module can't be used with RxJs v6. (npm update)

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem** via https://plnkr.co or similar.

Add module to a Angular 6 environment without rjxs-compat.
Then do npm install/update:

node_modules/ng2-toasty/src/toasty.service.d.ts(1,10): error TS2305: Module '"<redacted>/node_modules/rxjs/Observable"' has no exported member 'Observable'.

* **What is the expected behavior?**

module works

* **What is the motivation / use case for changing the behavior?**

see above

* **Please tell us about your environment:**

- Angular version: 6.0.0
- Browser: all

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, gitter, etc)

**Comment by Rzpeg** [comment]
module can be used with Angular 6, with rxjs-compat installed

**Comment by intuitivepixel** [comment]
> module can be used with Angular 6, with rxjs-compat installed

that is true I can confirm that it works, but are there any plans to update to rxjs 6 anytime soon?

**Comment by iamwyza** [comment]
@Rzpeg - Well yes, that can be said of alot of things, but that's a lousy answer.  -compat adds alot of bloat and, also, -compat will only work for rxjs 6.  rxjs7 will *not* have a -compat release. (the release notes for 6 stated this).  Out of all the modules I presently am using in my app, only this and one other don't properly handle this.  

So, the issue stands.

**Comment by Rzpeg** [comment]
This repo looks dead, no activity at all.

I have migrated my project to https://github.com/scttcper/ngx-toastr

**Comment by emonney** [comment]
I have ported this library here with angular6 and rxjs6 support. [ngx-toasta](https://github.com/emonney/ngx-toasta).
And also with some extra enhancements.

`npm install ngx-toasta`
